pyproject.toml

```diff
@@ -5,3 +5,8 @@ line-length = 119
 max-line-length = 119
 count = true
 exclude = "build,debian,dist,envs,fastentrypoints.py,__pycache__,usbmuxctl.egg-info,venv,contrib,setup.py,env-qa,env,env-packaging"
+
+[tool.isort]
+line_length = 119
+multi_line_output = 3
+include_trailing_comma = true
```

Comment:
Please also add the tool to the `qa`-target in the `Makefile`.

Reply:
Has been replaced by `ruff`, which does linting, formatting and import sorting and is now part of the `qa` target in the Makefile.